appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
I implemented your specifications in #96. Let me know if this addresses your request and, if so, we can merge both pull requests.